compiler/noirc_evaluator/src/ssa/parser/into_ssa.rs

```diff
@@ -156,10 +156,22 @@ impl Translator {
             RuntimeType::Brillig(inline_type) => {
                 self.builder.new_brillig_function(external_name, function_id, inline_type);
                 self.builder.set_globals(self.globals_graph.clone());
+
+                // In our ACIR generation tests we want to make sure that `brillig_locations` in the `GeneratedAcir` was accurately set.
+                // Thus, we set a dummy location here so that translated instructions have a location associated with them.
+                let stack = vec![Location::dummy()];
+                let call_stack = self
+                    .builder
+                    .current_function
+                    .dfg
+                    .call_stack_data
+                    .get_or_insert_locations(&stack);
```

Comment:
nit: just a few less lines to ease readability

```suggestion
                let call_stack_data = &mut self.builder.current_function.dfg.call_stack_data;
                let call_stack = call_stack_data.get_or_insert_locations(&stack);
```